spec.md

```diff
@@ -2606,6 +2742,24 @@ enumerate(["zero", "one", "two"])               # [(0, "zero"), (1, "one"), (2,
 enumerate(["one", "two"], 1)                    # [(1, "one"), (2, "two")]
 ```
 
+### float
+
+`float(x)` interprets its argument as a floating-point number.
+
+If x is a `float`, the result is x.
+
+If x is an `int`, the result is the floating-point value nearest x.
+The call fails if x is too large to represent as a `float`.
+
+If x is a `bool`, the result is `1.0` for `True` and `0.0` for `False`.
+
+If x is a string, the string is interpreted as a floating-point literal.
```

Comment:
Ah, so this is why `float(<crazy large int>)` fails with overflow but `float(<string repr of crazy large int>)` succeeds as inf. In that case, seems like we can't make float's behavior consistent unless we also make it an error to write crazy large float literals (which we should also specify the behavior of in this doc). I think I'd be up for making all those cases fail.

Reply:
Resolved that these cases all fail.